app/models/user.rb

```diff
@@ -1,6 +1,32 @@
 class User < ApplicationRecord
   ROLES = ['user', 'manager', 'admin']
 
+  class HeardAboutUs
```

Comment:
This inner class seems good to me. If we find that we want to change these we can always upgrade it to a Model later. Good ideal to put `{ additional: }` for the value instead of just a boolean. If were just a hash of String->Boolean it's easy to forget what the boolean represents and hard to add more features later.